manifests/init.pp

```diff
@@ -334,6 +340,9 @@
   Optional[String]  $ntsntpserver                                  = undef,
   Optional[Integer[0]] $ntsrotate                                  = undef,
   Optional[Integer[1,65535]] $acquisitionport                      = undef,
+  Optional[Stdlib::Absolutepath] $options_file                     = undef,
+  Optional[String] $options                                        = undef,
```

Comment:
Does it make sense to allow empty strings? Otherwise:
```suggestion
  Optional[String[1]] $options                                     = undef,
```